pkg/api/http_server.go

```diff
@@ -222,12 +220,12 @@ func ProvideHTTPServer(opts ServerOptions, cfg *setting.Cfg, routeRegister routi
 		queryDataService:             queryDataService,
 		serviceAccountsService:       serviceaccountsService,
 		authInfoService:              authInfoService,
-		TeamPermissionsService:       resourcePermissionServices.GetTeamService(),
 		NotificationService:          notificationService,
 		dashboardService:             dashboardService,
 		dashboardProvisioningService: dashboardProvisioningService,
 		folderService:                folderService,
 		DatasourcePermissionsService: datasourcePermissionsService,
+		TeamPermissionsService:       permissionsServices.GetTeamService(),
```

Comment:
I think we can make it private:
```suggestion
		teamPermissionsService:       permissionsServices.GetTeamService(),
```